cmd/fyne/internal/util/file.go

```diff
@@ -39,6 +39,31 @@ func EnsureSubDir(parent, name string) string {
 	return path
 }
 
+// EnsureAbsPath returns the absolute path of the given file if it is not already absolute
+func EnsureAbsPath(path string) string {
+	if !filepath.IsAbs(path) {
```

Comment:
We prefer to exit early when we can instead of having method contents indented in long ifs.
i.e.
```go
if file path.IsAbs(path) {
    return path
}
```